radar/2023-09-12/kubernetes.md

```diff
@@ -5,4 +5,5 @@ quadrant:   tools
 tags:       [infrastructure]
 ---
 
-Only needed for Medic hosting right now.
\ No newline at end of file
+Kubernetes is only needed for Medic hosting right now.
```

Comment:
We're promoting k3s/k3d for self hosting too, which is k8s compatible, so I think it'd be better to say...

```suggestion
Kubernetes platform is used for hosting the CHT. For self hosting partners we recommend lightweight implementations like k3d. For cloud deployments, use a Platform as a Service offering like Amazon EKS.
```

Might still be too technical but you get the idea...